packages/babel-plugin-polyfill-corejs3/src/index.ts

```diff
@@ -54,7 +52,7 @@ export default defineProvider<Options>(function (
     version = 3,
     proposals,
     shippedProposals,
-    [runtimeCompat]: { useBabelRuntime, ext = ".js" } = {},
+    [runtimeCompat]: { useBabelRuntime, ext = ".js" } = { useBabelRuntime: "" },
```

Comment:
Per [prev discussion](https://github.com/babel/babel-polyfills/pull/121#discussion_r837995855), left `ext = ".js"` in in this file and the other as it was already there. Let me know if you'd like me to change this

Reply:
This is all legacy code that hopefully will be removed at some point in the future; let's leave this as-is :grimacing: